src/transdialog.h

```diff
@@ -44,15 +44,13 @@ class mmTransDialog : public wxDialog
     mmTransDialog() {}
     virtual ~mmTransDialog();
 
-    mmTransDialog(
-        wxWindow* parent
+    mmTransDialog(wxWindow* parent
         , int account_id
         , int transaction_id
         , double current_balance
         , bool duplicate = false
         , int type = Model_Checking::WITHDRAWAL
-        , const wxString& name = "Transactions Dialog"
-    );
+        , const wxString& = "Transactions Dialog");
```

Comment:
1. Should this be translatable?
2. If so, should `Dialog` be removed? https://github.com/moneymanagerex/moneymanagerex/issues/5881

Thank you

Reply:
The string "Transactions Dialog" does not appear in the UI.

It is shown to the user as "New Transaction" or "Edit Transaction"

Reply:
The variable name is not used at all by the constructor, so it can be removed.